src/arch/xtensa/xtos/xtos-structs.h

```diff
@@ -55,6 +56,7 @@ struct core_context {
 	struct schedulers *schedulers;
 	struct notify *notify;
 	struct idc *idc;
+	struct sof_core_context *sof;
```

Comment:
I don't quite understand this change. Why do we add another core context to core context?

Reply:
I didn't feel quite right to have audio/component/pipeline-related things in here, so I wanted to move it to a separate structure.

Reply:
it's probably a separate patch anyways?